_pytest/python.py

```diff
@@ -131,6 +131,7 @@ def pytest_namespace():
     raises.Exception = pytest.fail.Exception
     return {
         'raises': raises,
+        'not_raises': NotRaisesContext,
```

Comment:
> we should either clearly limit it, or clearly support passing exception types from the get go

OK I don't have a strong objection here.

Also asked in the mailing list if anyone sees a problem in making this a singleton. 👍